src/network/Connection.ts

```diff
@@ -79,17 +79,28 @@ export class PipelinedWriter extends Writer {
         });
     }
 
+    private isClosed(): boolean {
```

Comment:
Not sure how things are optimized in Node.js but are method calls expensive here? I mean, if they are, I think the old way of doing things (with the name `error` instead of `closeCause`) seems OK to me

Reply:
Ok removed the method and renamed the variable